### DISCUSSION TITLE
Occational stuttering on all transports

### DISCUSSION THREAD

**Discussion by agg23** [bluenviron/mediamtx#2223] [Questions] [answered]
I've been trying for the past few days to debug a series of issues preventing me from using `mediamtx`. I have tried:

* Direct streaming HLS sources
* Passing HLS sources through machine local `ffmpeg` -> RTSP
* Passing HLS sources through remote `ffmpeg` -> RTSP
* Passing local media through `ffmpeg` -> RTSP

All of these have resulted in occasional (every 30s - 1min) stuttering for a second or more on RTSP and WebRTC outputs. The stutters do not occur at the same time across transports, but within a given transport (say WebRTC), they drop at the same time. This suggests to me something occurring serverside, and not a packet loss issue.

I don't believe the HLS sources at least are too high of bitrate (because they're streamed on their own normally), and considering I'm currently testing within my LAN, I would not expect even the local media to be stuttering.

`mediamtx` debug logs are empty during this time, and really only contain the client connection/disconnection messages.

Thank you for any suggestions

**Comment by aler9** [comment]
Open an issue and provide enough details to replicate the situation, including a test source and the name of the client that is reading the stream.

**Reply by agg23** [reply, marked as answer]
Created #2227

[a comment by an automated account is not shown]
